packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/explore/TablesViewer.tsx

```diff
@@ -48,6 +49,7 @@ export function TablesViewer({ table, query }: { table?: TableType; query?: stri
               className="-ml-4"
               onClick={() => column.toggleSorting(column.getIsSorted() === "asc")}
             >
+              {Object.keys(keySchema).includes(name) && <KeyIcon className="mr-2 h-3 w-3" />}
```

Comment:
you could do `keySchema[name]` or `name in keySchema` here to avoid converting this to an array then iterating through it

you could also `table.schema.key.includes(name)`, as in no need to generate a key schema from the table since the schema already lists the key field names

Reply:
oh for real, nicely spotted! not sure why I did it that way, used `name in keySchema` in the end, like the descriptiveness of it.